### PR TITLE
Initialize workspace

### LINKED ISSUE #1: Set up a multi-project workspace with `pnpm`
Which is a fancy way of saying "create [`pnpm-workspace.yaml`](https://pnpm.io/pnpm-workspace_yaml)". Create empty projects for the podcast app and the presentation.

**Acceptance criteria**:
- `pnpm` detects that there are two projects in the workspace
- The projects are named `app` and `presentation`
- Each project contains a `package.json`

### PR DESCRIPTION
**Pull request by VanishMax**
Closes #1

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "cast",
+  "version": "0.1.0",
+  "description": "A podcast listening PWA app",
```
Comment (illright) [inline app/package.json:4]: [nitpick (non-blocking)](https://conventionalcomments.org/#labels): "PWA app" repeats the word "app" twice

File: pnpm-workspace.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+packages:
+  - app
+  - presentations
```
Comment (illright) [inline pnpm-workspace.yaml:3]: [issue](https://conventionalcomments.org/#labels): I don't think we're going to have multiple presentations, rather that presentation 2 is an outgrown version of presentation 1
[suggestion](https://conventionalcomments.org/#labels): let's name it `presentation`

**Review by illright — CHANGES_REQUESTED** [review]

**Comment by illright** [comment]
Requested minor changes with respect to the acceptance criteria

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "cast-monorepo",
+  "version": "0.1.0",
+  "description": "A monorepo for the Cast app",
+  "private": true,
+  "scripts": {
+    "preinstall": "npx only-allow pnpm"
+  },
+  "repository": {
```
Comment (illright) [inline package.json:9]: [praise](https://conventionalcomments.org/#labels): this is quite an extensive `package.json`, cool :)

File: pnpm-workspace.yaml (hunk 2 above)
Comment (VanishMax) [reply pnpm-workspace.yaml]: fixed

File: app/package.json (hunk 1 above)
Comment (VanishMax) [reply app/package.json]: fixed

**Review by illright — APPROVED** [review]

**Pull request merged**